app/src/main/java/com/example/androidfundamental/MyAdapter.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.androidfundamental
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class MyAdapter(var values: MutableList<String>) : RecyclerView.Adapter<MyAdapter.MyViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MyViewHolder {
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.row_layout, parent, false)
+        return MyViewHolder(view)
+    }
+
+    override fun getItemCount(): Int {
+        return values.size
+    }
+
+    override fun onBindViewHolder(holder: MyViewHolder, position: Int) {
+        val string = values.get(position)
+        holder.bind(string)
+    }
+
+    fun removeAt(position: Int) {
+        values.removeAt(position)
+        notifyItemRemoved(position)
+    }
+
+    inner class MyViewHolder(private val view: View) : RecyclerView.ViewHolder(view) {
+        fun bind(string: String) {
```

Comment:
You could reuse the id like:

![image](https://user-images.githubusercontent.com/36905736/72947139-bb716f80-3d46-11ea-93b2-48614b8045b7.png)